lib/protoflow-blocks/src/block_tag.rs

```diff
@@ -46,6 +46,8 @@ pub enum BlockTag {
     // TextBlocks
     ConcatStrings,
     SplitString,
+    DecodeCsv,
+    EncodeCsv,
```

Comment:
These must always be in alphabetical order

Reply:
It looks like it should be under // TextBlocks not in alphabetical order

Reply:
Or should make a change here to alphabetical order also?

Reply:
At least have everything under `TextBlocks` in alphabetical order

Reply:
got it